src/main/java/uk/gov/companieshouse/charges/data/config/WebSecurityConfig.java

```diff
@@ -27,11 +32,13 @@ public SecurityFilterChain filterChain(HttpSecurity http, Logger logger) throws
                 .csrf(AbstractHttpConfigurer::disable)
                 .formLogin(AbstractHttpConfigurer::disable)
                 .logout(AbstractHttpConfigurer::disable)
+                .cors(AbstractHttpConfigurer::disable)  // Added CORS disable
```

Comment:
I don't think the comments here and L38 add anything

Reply:
removed 👍